app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -8,6 +8,16 @@ module WorksControllerBehavior
       layout :decide_layout
     end
 
+    class_methods do
+      # We don't want the breadcrumb action to occur until after the concern has
+      # been loaded and authorized
+      def curation_concern_type=(curation_concern_type)
+        super
+        before_action :build_breadcrumbs, only: [:edit, :show, :new]
```

Comment:
Sorry for the delay, I've been between conferences.

I'm currently working through (https://gist.github.com/jrgriffiniii/25234e81cea04d96073407ff297b5d2e) and will determine why this raises
`Nil location provided. Can't build URI.`
for
```ruby
main_app.polymorphic_path(curation_concern)
```